corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -64,6 +64,25 @@ def _get_case_case_counts_by_owner(domain, datespan, case_types, is_total=False)
     return case_query.run().aggregations.owner_id.counts_by_bucket()
 
 
+def get_active_case_count(domain, datespan, case_types, is_total=False):
```

Comment:
any chance we can get a test for this? we have a pretty extensive suite going here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/tests/test_esaccessors.py. reports are one of those areas in our code base that doesn't have a lot of coverage so at least making sure the database access is tested is good!